CHANGELOG.rst

```diff
@@ -233,7 +233,11 @@ time or change existing behaviors in order to make them less surprising/more use
   Thanks `@Vogtinator`_ for reporting and `@RedBeardCode`_ and
   `@tomviner`_ for the PR.
 
-*
+* Added confcutdir in testing/test_conftest.py::test_conftest_import_order and
```

Comment:
TBH as it stands, I'm not sure this change should be in the CHANGELOG at all... it is a small change in two of pytest's internal tests.

On the other hand if we go with my suggestion to change the `testdir` fixture to also set a `confcut` directory, then a changelog entry is warranted because it may affect plugin authors.